packages/react-native-renderer/src/ReactNativeFiberRenderer.js

```diff
@@ -155,12 +176,15 @@ const NativeRenderer = ReactFiberReconciler({
     return false;
   },
 
-  getRootHostContext(): {} {
-    return emptyObject;
+  getRootHostContext(rootContainerInstance: Container): HostContext {
+    return {type: 'root'};
   },
 
-  getChildHostContext(): {} {
-    return emptyObject;
+  getChildHostContext(
+    parentHostContext: HostContext,
+    type: string,
+  ): HostContext {
+    return {type};
```

Comment:
If you don't need this in PROD, let's avoid the extra allocation? Like ReactDOM does.

Reply:
Same for the root one.